conf/app.routes

```diff
@@ -92,10 +92,10 @@ POST        /create-record/goods-description-question/check  controllers.UseTrad
 GET         /create-record/cya-create-record                 controllers.CyaCreateRecordController.onPageLoad
 POST        /create-record/cya-create-record                 controllers.CyaCreateRecordController.onSubmit
 
-GET         /trader-reference                                controllers.TraderReferenceController.onPageLoad(mode: Mode = NormalMode)
-POST        /trader-reference                                controllers.TraderReferenceController.onSubmit(mode: Mode = NormalMode)
-GET         /trader-reference/check                          controllers.TraderReferenceController.onPageLoad(mode: Mode = CheckMode)
-POST        /trader-reference/check                          controllers.TraderReferenceController.onSubmit(mode: Mode = CheckMode)
+GET         /create/trader-reference                         controllers.TraderReferenceController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
shouldn't this be create-record instead of create?

Reply:
I raised this with Amaka. She asked me to stick what confluence has for now https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=861962859 :(